### ISSUE TITLE
Does this supports SMPLX format ?

### ISSUE THREAD

**Issue by SreehariRajan** [LinghaoChan/OpenTMA#5]
Does the training code support smplx format?

**Comment by LinghaoChan** [comment]
yes!

**Comment by SreehariRajan** [comment]
Could you give the instructions to train temos in smplx format ?

**Comment by LinghaoChan** [comment]
I provide an example of motion-x based training command, where motion-x is smpl-x data format.

**Comment by SreehariRajan** [comment]
But motion-x pose representation uses 623 dim vectors and for smplx it's only 322 right ?

**Comment by LinghaoChan** [comment]
exactly. You can directly change the shape of the motion representation during training. That. is to say, you only need to change the hyper-parameter $D$ (motion $\in \mathbb{R}^{D\times T}$)

**Comment by SreehariRajan** [comment]
So we don't have to change anything in MotionX-TMR.yaml right (keeping MOTION_TYPE as vector_623)? We just need to give the input motion pose as 322 dim.

**Comment by SreehariRajan** [comment]
So we don't have to change anything in MotionX-TMR.yaml right (keeping MOTION_TYPE as vector_623)? We just need to give the input motion pose as 322 dim ?

**Comment by SreehariRajan** [comment]
How can we train temos with pose representation in rotation form instead of joint locations?
